src/components/Table/TableHeader/TableHeader.tsx

```diff
@@ -1,24 +1,34 @@
-import React, { forwardRef } from "react";
+import React, { forwardRef, useMemo } from "react";
 import { VibeComponent, VibeComponentProps } from "../../../types";
 import styles from "./TableHeader.module.scss";
 import { ITableHeaderCellProps } from "../TableHeaderCell/TableHeaderCell";
 import cx from "classnames";
 import { getTestId } from "../../../tests/test-ids-utils";
 import { ComponentDefaultTestId } from "../../../tests/constants";
+import { RowSizes } from "../Table/TableConsts";
 
 export interface ITableHeaderProps extends VibeComponentProps {
   children?: React.ReactElement<ITableHeaderCellProps> | React.ReactElement<ITableHeaderCellProps>[];
+  size?: RowSizes;
 }
 
 const TableHeader: VibeComponent<ITableHeaderProps, HTMLDivElement> = forwardRef(
-  ({ id, className, "data-testid": dataTestId, children }, ref) => {
+  ({ id, className, "data-testid": dataTestId, children, size = RowSizes.MEDIUM }, ref) => {
+    const sizeStyle = useMemo(() => {
```

Comment:
It was not sure if it was worth to extract this logic to some generic component but eventually decided against it. What do you think?

Reply:
I suggest to use classnames and not inline styles. conditionally set the appropriate classname based on the size (small, medium)

Reply:
I have re-implemented it